app/models/github/task.rb

```diff
@@ -0,0 +1,62 @@
+class Github::Task < ApplicationRecord
+  disable_sti!
+
+  belongs_to :track, optional: true
+
+  enum action: {
+    create: 0,
+    fix: 1,
+    improve: 2,
+    proofread: 3,
+    sync: 4
+  }, _suffix: true
```

Comment:
I added the suffix as rails would otherwise complain about the "create" method already being defined